app/controllers/api/V1/submissions_controller.rb

```diff
@@ -115,9 +104,9 @@ def completed_but_no_attachment?
         submission.status.eql?('completed') && attachment.nil?
       end
 
-      def result_or_status
-        submission.status.eql?('completed') ? 'result' : 'status'
-      end
+      # def result_or_status
```

Comment:
can this commented out block be removed?

Reply:
Good catch - I couldn't see any other uses and wanted to check, then forgot to delete it!